butterfree/load/writers/historical_feature_store_writer.py

```diff
@@ -232,7 +232,8 @@ def validate(
         )
         written_count = (
```

Comment:
I think that you could establish a condition for full and interval loads since you won't need to define the partitions when performing a full load, you just need the path provided by the method ```get_options``` within ```S3Config```.

Reply:
The forms of validation are different for the full and the interval, when we do a full job it will validate using the `read_table` method and when it is an interval job it will validate through the partitions.